themes/startwords/README.md

```diff
@@ -15,6 +15,7 @@ The Startwords Hugo theme is designed for [a journal of the same name](https://s
 - Article order in an issue is configurable. The first two articles will be displayed as featured essays highlighted side by side on the issue page; all other articles will be listed by title in the order specified.
 - Layouts: custom content types include feature articles [available under [layouts/article](https://github.com/Princeton-CDH/startwords/tree/master/themes/startwords/layouts/article)], single issues, and list of issues [both available under [layouts/issue](https://github.com/Princeton-CDH/startwords/tree/master/themes/startwords/layouts/issue)].
 - To retain the simplest directory structure that will give us the URLs we want, articles are placed in an issue number directory and have `type:article` specified in the page metadata.
+- The number of features in a given issue can be selected with the `feature_count` parameter. It's default value is 2.
```

Comment:
Can we rename `feature_count` to `num_features` ? That seems more obvious to me. 

Suggested rewording.

If there is a limit to what we support or have tested, that's probably worth noting here too.

```suggestion
- The feature essays for an issue are determined by order and the number of features in that issue (the features come first). By default, each issue has two features, but that can be customized by setting `num_features` in the issue metadata.  
```